src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -8,6 +8,16 @@
 
 namespace System
 {
+   public partial class AppDomain
```

Comment:
At this point I was just asking the question to see why it was chosen, but I'm not sure were we should put it yet. There is a decent chance that everything in System.Runtime.Extensions gets folded into System.Runtime and when that happens I'm not sure I want AppDomain to be part of that. So we would have to keep it here or separate it at that point. I'm wondering if we shouldn't just have a System.AppDomain library that is independent.